src/extension.ts

```diff
@@ -150,6 +155,8 @@ function registerEnvironmentVariableContributions(): void {
         actionContext.telemetry.suppressAll = true;
         actionContext.errorHandling.suppressDisplay = true;
 
+        logDockerEnvironment(ext.outputChannel);
```

Comment:
I'd move this line down into the `if` below, since this event gets called for _all_ settings changes. It could become quite noisy.